File: Modul 2. Input Output/PRAK205/PRAK205-2310817120003-Natalie_Grace_Katiandagho.py
import math
berapa = input()
apa_ya = berapa.split()
a, b = map(int, apa_ya)

c = int(math.sqrt(b ** 2 - a ** 2)) 
K = int(a + b + c)
L = int( 0.5 * c * a)

print(f'Alas = {c} cm')
print(f'Tinggi = {a} cm')
print(f'Keliling = {K} cm')
print(f'Luas = {L} cm^2')

File: Modul 2. Input Output/PRAK205/PRAK205-2310817120003-Natalie_Grace_Katiandagho.c
#include <stdio.h>
#include <math.h>
void main()
{
    float a;
    float b;

    scanf("%f %f", &a, &b);
    float c = sqrt(b * b - a * a);
    float K = a + b + c;
    float L = a * c / 2;
    printf("Alas = %.0f cm\n", c);
    printf("Tinggi = %.0f cm\n", a);
    printf("Keliling = %.0f cm\n", K);
    printf("Luas = %.0f cm^2\n", L);
}